apps/mobile/src/screens/chat/libs/components/loader/libs/helpers/get-animated-styles.helper.ts

```diff
@@ -0,0 +1,12 @@
+import { type SharedValue, useAnimatedStyle } from "react-native-reanimated";
+
+type Property = {
+	transform: { scale: number }[];
+};
+
+const getAnimatedStyle = (scaleValue: SharedValue<number>): Property =>
```

Comment:
Let's convert it to a custom hook. Hooks cannot be called inside of helpers
![image](https://github.com/user-attachments/assets/60e49baa-181d-46b2-95cb-ba09d366eaab)